src/main/java/seedu/address/logic/commands/AddInterviewCommand.java

```diff
@@ -143,6 +143,7 @@ public CommandResult execute(Model model) throws CommandException {
 
 
         model.addInterview(interview);
+        model.sortInterview();
```

Comment:
Nice call on sorting everytime an interview is added so the user doesn't have to execute a separate command to sort the interviews. This also helps users see at all times which interview is first